packages/flutter/lib/src/material/menu_anchor.dart

```diff
@@ -923,6 +924,18 @@ class MenuItemButton extends StatefulWidget {
   /// {@endtemplate}
   final bool closeOnActivate;
 
+  /// The direction in which the menu item expands.
+  ///
+  /// If the menu item button is created by the [MenuAnchor] or [MenuBar], then
```

Comment:
The `MenuItemButton` not really created by them, but it can appear as their child. Might want to reword this.

Reply:
Good point!